news/2019-06-15-new-featured-artist-riran.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "New Featured Artist: RiraN"
+date: 2019-06-15 06:00:00 +0000
+---
+
+osu!'s latest Featured Artist enters the scene in style - hardstyle infact! Meet [**RiraN**](https://osu.ppy.sh/beatmaps/artists/40), one of Asia's up and coming hardstyle artists!
```

Comment:
First sentence reads weirdly. Was it meant to be a joke about "entering the scene *hard*style" or something?

Also "infact" is not really a word!

Reply:
it was yes and you're correct, infact isn't a word, in fact